README.md

```diff
@@ -26,6 +26,10 @@ Ensure that new API versions don't break API consumers with breaking change dete
 
 ![Generate Open API specification](./media/generate-api-documentation.gif)
 
+[[Pre-Release]](#pre-release)
+API Center Data View

```

Comment:
I think this feature is worthy to be a seperate section.
@sthirthala thoughts?

Reply:
Yes makes sense to have separate section